org.flathub.flatpak-external-data-checker.yml

```diff
@@ -214,12 +214,13 @@ modules:
           - name: libsodium
             sources:
               - type: archive
-                url: https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz
-                sha256: 6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1
+                url: https://github.com/jedisct1/libsodium/archive/1.0.18/libsodium/1.0.18.tar.gz
+                sha256: d59323c6b712a1519a5daf710b68f5e7fde57040845ffec53850911f10a5d4f4
                 x-checker-data:
                   type: html
```

Comment:
Since we're switching to github sources, it makes sense to track github releases using json checker.
```yaml
x-checker-data:
  type: json
  url: https://api.github.com/repos/jedisct1/libsodium/releases/latest
  tag-query: .tag_name
  version-query: $tag | sub("-RELEASE$"; "")
  url-query: >-
    .assets | map(select(.name=="libsodium-\($version).tar.gz")) | first | .browser_download_url
```